src/_pytest/hookspec.py

```diff
@@ -324,6 +321,7 @@ def pytest_ignore_collect(
     Stops at first non-None result, see :ref:`firstresult`.
 
     :param collection_path: The path to analyze.
+    :type collection_path: pathlib.Path
```

Comment:
Why are those (also in other files) needed? I thought Sphinx could interpret the type annotations, no?

Reply:
it doesnt :scream_cat: